src/python/gudhi/delaunay_complex.pyx

```diff
@@ -18,6 +19,7 @@ from libcpp.string cimport string
 from libcpp cimport bool
 from libc.stdint cimport intptr_t
 import warnings
+from typing import Literal, Optional, Iterable
```

Comment:
I saw that `typing.Iterable` was [deprecated](https://docs.python.org/3/library/typing.html#typing.Iterator) (and aliased to `collections.abc`) 
```suggestion
from typing import Literal, Optional
from collections.abc import Iterable
```